tools/gitlab.md

```diff
@@ -0,0 +1,64 @@
+---
+permalink: /gitlab
+layout: post
+title: GitLab
+command: gitlab-rake gitlab:env:info
+link: https://docs.gitlab.com/ce/policy/maintenance.html
+changelogTemplate: https://gitlab.com/gitlab-org/gitlab/-/releases/v__RELEASE_CYCLE__.0-ee
```

Comment:
~Correct if I'm wrong, but this covers changelog for _all releases_, not just EE right?~

Edit: There are [different changelogs](https://about.gitlab.com/releases/) but this should suffice.

Reply:
[This changelog](https://gitlab.com/gitlab-org/gitlab/-/releases) from GitLab releases merge all changes regarding GitLab's subscription (Core, Premium or Ultimate). So, I think it's the better source to track changelogs for all kind of subscription.